kernelci.org/content/en/docs/team/board.md

```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
 ---
 
 The Advisory Board (AB) is composed of one representative from each premium
-member organization.  The board elects one Chair and one Treasurer each year.
-A budget is agreed each year and voted by the board.
+member organization, one representative for every five general member
+organizations (maximum of three) and the TSC Chair.
+
+The board elects one Chair and one Treasurer each year. A budget is agreed each
+year and voted by the board.
+
+There is also a Program Manager role to help coordinating things with the wider
```

Comment:
```suggestion
There is also a Project Manager role to help coordinating things with the wider
```
Kevin's role was called Program Manager but responsibilities have now been split a new Project Manager role and the Chair.